umbral/pre.py

```diff
@@ -260,15 +264,16 @@ def __eq__(self, other):
             # Again, it's hard to imagine why.
             return False
 
-    def __hash__(self):
+    @typing.no_type_check
```

Comment:
Just curious, why?

Reply:
there is a tricky check inside the genexp here.